### PR TITLE
add us w9

### PR DESCRIPTION
**Pull request by ianardee**

### CODE REVIEW

File: products/us_w9/response_v1/summary_page0.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+:SSN: 560758145
+:Address: Somewhere In Milky Way
+:City State Zip: Probably Still At Cambridge P O Box CB1
+:Business Name:
+:EIN: 942203664
+:Tax Classification: individual
+:Tax Classification Other Details:
+:W9 Revision Date: august 2013
+:Signature Position: Polygon with 4 points.
+:Signature Date Position: Polygon with 0 points.
```
[the remark this answers is not in the document]
Comment (ianardee) [reply products/us_w9/response_v1/summary_page0.rst:13]: It's technically correct, as the empty return is `[]` and not `null`.

[the remark this answers is not in the document]
Comment (ianardee) [reply products/us_w9/response_v1/summary_page0.rst:13]: But we could change the string output to return "".

[the remark this answers is not in the document]
Comment (sebastianMindee) [reply products/us_w9/response_v1/summary_page0.rst:13]: Yeah that sounds better to me. Plus, a **poly**gon should have **several** vertices :upside_down_face:

[the remark this answers is not in the document]
Comment (ianardee) [reply products/us_w9/response_v1/summary_page0.rst]: done

**Review by sebastianMindee — APPROVED** [review]
:D

**Pull request merged**